gp-includes/misc.php

```diff
@@ -63,7 +63,7 @@ function gp_route_nonce_url( $url, $action ) {
  * @return mixed $array[$key] if exists or $default
  */
 function gp_array_get( $array, $key, $default = '' ) {
-	return isset( $array[ $key ] ) ? $array[ $key ] : $default;
+	return ( is_string( $key ) || is_numeric( $key ) ) && isset( $array[ $key ] ) ? $array[ $key ] : $default;
```

Comment:
You could probably use `is_scalar()` instead if you wanted something shorter:

A personal preference is to use brackets around the terninary operator when multiple function calls are present too:

```suggestion
	return ( is_scalar( $key ) && isset( $array[ $key ] ) ) ? $array[ $key ] : $default;
```